### PR TITLE
Add Component.propTypes

### PR DESCRIPTION
**Pull request by cesargdm**
Closes https://github.com/meedan/check/issues/19

### CODE REVIEW

File: src/app/components/ExternalLink.js [hunk 1]
```diff
@@ -1,9 +1,20 @@
 import React from 'react';
+import PropTypes from 'prop-types'
```
Comment (amoedoamorim) [inline src/app/components/ExternalLink.js:2]: Please add a semicolon

File: src/app/components/ExternalLink.js [hunk 2]
```diff
@@ -1,9 +1,20 @@
...
 const ExternalLink = props => (
   <a href={props.url} style={props.style} target="_blank" rel="noopener noreferrer">
     {props.children}
   </a>
 );
 
+ExternalLink.propTypes = {
+  children: PropTypes.node.isRequired,
+  style: PropTypes.object,
+  url: PropTypes.string.isRequired
```
Comment (amoedoamorim) [inline src/app/components/ExternalLink.js:13]: Please add a trailing comma

File: src/app/components/ExternalLink.js [hunk 3]
```diff
@@ -1,9 +1,20 @@
...
+}
```
Comment (amoedoamorim) [inline src/app/components/ExternalLink.js:14]: Please add a semicolon

File: src/app/components/ExternalLink.js [hunk 4]
```diff
@@ -1,9 +1,20 @@
...
+
+ExternalLink.defaultProps = {
+  style: {}
```
Comment (amoedoamorim) [inline src/app/components/ExternalLink.js:17]: Please add a trailing comma

File: src/app/components/ExternalLink.js [hunk 5]
```diff
@@ -1,9 +1,20 @@
...
+}
```
Comment (amoedoamorim) [inline src/app/components/ExternalLink.js:18]: Please add a semicolon

File: src/app/components/UserTosForm.js [hunk 6]
```diff
@@ -1,4 +1,5 @@
 import React, { Component } from 'react';
+import PropTypes from 'prop-types'
```
Comment (amoedoamorim) [inline src/app/components/UserTosForm.js:2]: Please add a semicolon

File: src/app/components/Message.js [hunk 7]
```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+import PropTypes from 'prop-types'
```
Comment (amoedoamorim) [inline src/app/components/Message.js:2]: Please add a semicolon

File: src/app/components/Message.js [hunk 8]
```diff
@@ -51,4 +52,13 @@ const Message = (props) => {
   return null;
 };
 
+Message.propTypes = {
+  message: PropTypes.string,
+  onClick: PropTypes.func.isRequired
```
Comment (amoedoamorim) [inline src/app/components/Message.js:57]: Please add a trailing comma

File: src/app/components/Message.js [hunk 9]
```diff
@@ -51,4 +52,13 @@ const Message = (props) => {
...
+}
```
Comment (amoedoamorim) [inline src/app/components/Message.js:58]: Please add a semicolon

File: src/app/components/Message.js [hunk 10]
```diff
@@ -51,4 +52,13 @@ const Message = (props) => {
...
+
+Message.defaultProps = {
+  message: ''
```
Comment (amoedoamorim) [inline src/app/components/Message.js:61]: Please add a trailing comma

File: src/app/components/Message.js [hunk 11]
```diff
@@ -51,4 +52,13 @@ const Message = (props) => {
...
+}
```
Comment (amoedoamorim) [inline src/app/components/Message.js:62]: Please add a semicolon

File: src/app/components/UserTosForm.js [hunk 12]
```diff
@@ -132,4 +133,20 @@ class UserTosForm extends Component {
...
 
+UserTosForm.propTypes = {
+  checkedPp: PropTypes.bool.isRequired,
+  checkedTos: PropTypes.bool.isRequired,
+  handleCheckPp: PropTypes.func.isRequired,
+  handleCheckTos: PropTypes.func.isRequired,
+  showTitle: PropTypes.bool,
+  termsLastUpdatedAt: PropTypes.number.isRequired,
+  user: PropTypes.shape({
+    last_accepted_terms_at: PropTypes.number
```
Comment (amoedoamorim) [inline src/app/components/UserTosForm.js:144]: Please change type to string and also add a trailing comma

File: src/app/components/UserTosForm.js [hunk 13]
```diff
@@ -132,4 +133,20 @@ class UserTosForm extends Component {
...
+  }).isRequired,
+};
+
+UserTosForm.defaultProps = {
+  showTitle: false
```
Comment (amoedoamorim) [inline src/app/components/UserTosForm.js:149]: Please add a trailing comma

File: src/app/components/Message.js [hunk 14]
```diff
@@ -51,4 +52,13 @@ const Message = (props) => {
   return null;
 };
 
+Message.propTypes = {
+  message: PropTypes.string,
```
Comment (amoedoamorim) [inline src/app/components/Message.js:56]: Please make it also accept a PropTypes.node

**Review by amoedoamorim — CHANGES_REQUESTED** [review]
Hi @cesargdm ! Thank you very much for your contribution! Your PR looks great! I just need to ask you to do a few changes so we can accept and merge it! Again, thanks for participating!

File: src/app/components/UserTosForm.js [hunk 15]
```diff
@@ -132,4 +133,20 @@ class UserTosForm extends Component {
   }
 }
 
+UserTosForm.propTypes = {
+  checkedPp: PropTypes.bool.isRequired,
+  checkedTos: PropTypes.bool.isRequired,
+  handleCheckPp: PropTypes.func.isRequired,
+  handleCheckTos: PropTypes.func.isRequired,
+  showTitle: PropTypes.bool,
+  termsLastUpdatedAt: PropTypes.string.isRequired,
```
Comment (amoedoamorim) [inline src/app/components/UserTosForm.js:142]: Please revert this back to number

File: src/app/components/UserTosForm.js [hunk 16]
```diff
@@ -132,4 +133,20 @@ class UserTosForm extends Component {
...
+  user: PropTypes.shape({
+    last_accepted_terms_at: PropTypes.number,
```
Comment (amoedoamorim) [inline src/app/components/UserTosForm.js:144]: Please change this type to string

**Review by amoedoamorim — CHANGES_REQUESTED** [review]
Thanks again, @cesargdm ! We appreciate your contribution!
There's just one error that needs to be fixed before we can accept your PR.
`termsLastUpdatedAt` should be kept as `number`. What needs to be changed is `user.last_accepted_terms_at` because it's returned as a string from the backend!

**Comment by cesargdm** [comment]
> Thanks again, @cesargdm ! We appreciate your contribution!
> There's just one error that needs to be fixed before we can accept your PR.
> `termsLastUpdatedAt` should be kept as `number`. What needs to be changed is `user.last_accepted_terms_at` because it's returned as a string from the backend!

Sure, thanks

**Review by amoedoamorim — APPROVED** [review]
Yay! Congrats on your first approved PR on Check! :tada: :100:

**Comment by cesargdm** [comment]
> Yay! Congrats on your first approved PR on Check! 🎉 💯

Hey have you considered adding a prettierrc or/and editorconfig?

**Comment by amoedoamorim** [comment]
I'm not sure if we have considered those before but we do use eslint to enforce some coding standards/guidelines. Will have a look at them, thanks for sharing!

**Pull request merged**